blocks/form/form.js

```diff
@@ -1,13 +1,15 @@
 import { extractFormDefinition as extractSheetDefinition, renderForm as renderDocForm } from './lib/docform.js';
 import decorateUTM from './utm.js';
-import { buildBlock, getMetadata } from '../../scripts/aem.js';
+import { sendDigitalDataEvent } from '../../scripts/martech.js';
+import { getFormName } from './utils.js';
 
 export default async function decorate(block) {
   const { container, formDef } = await extractSheetDefinition(block);
   if (formDef && formDef.properties?.source === 'sheet') {
     const form = await renderDocForm(formDef);
     await decorateUTM(form);
     if (form) {
+      form.setAttribute('tabindex', '-1');
```

Comment:
is it for giving highlight priority for form focus on tab?

Reply:
In live site when modal is opened the formStart event is sent to analytics, I have a listener for sending this on `focusin` event. That's why this is here